### ISSUE TITLE
Document the DAMS stack in light of our updated environment

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/ucsc-library-digital-collections#454]
### Descriptive summary
Prior to his departure, Ned produced a [diagram of the DAMS stack](https://docs.google.com/drawings/d/1FFnJ9etaBRdvRei2yduo_w7lppJ5M4TNhnsjUrtdJS8/edit) as well as a [reference guide to related documentation](
https://docs.google.com/document/d/10yZYgOzd7DY2BBJaDf82v5J8a2HRH37eoG_kp3koKSY/edit#heading=h.9j8ivys70oy).  We've determine that these documents need to be reviewed and updated/re-created to represent the DAMS stack as it is at present.

### Acceptance criteria
**Done means:**
- [ ] DAMS stack diagram has been reviewed and revised
- [ ] DAMS stack reference guide has been reviewed and updated

**Comment by rmjaffe** [comment]
@snehagunduraoUL After meeting with Tania, it sounds like this has already been partially done -- i.e., the reference guide has been updated.

**Comment by snehagunduraoUL** [comment]
Yes. That's the same documentation work that I mentioned in today's standup. Please let me know if you need more info to be added to it.

**Comment by rmjaffe** [comment]
At this point, is it necessary to update the diagram or should we remove that criterion, put some points on the ticket and close it out?

**Comment by snehagunduraoUL** [comment]
Since we are at it, it makes sense to update the diagram as well. I have some room left tomorrow. I'll update it and share with the team.

**Comment by rmjaffe** [comment]
@snehagunduraoUL Thank you!!
